Libraries/Opc.Ua.PubSub/Encoding/UadpNetworkMessage.cs

```diff
@@ -371,7 +371,7 @@ public override byte[] Encode(IServiceMessageContext messageContext)
         /// <param name="stream">The stream to use.</param>
         public override void Encode(IServiceMessageContext messageContext, Stream stream)
         {
-            using (BinaryEncoder encoder = new BinaryEncoder(stream, messageContext))
+            using (BinaryEncoder encoder = new BinaryEncoder(stream, messageContext, true))
             {
                 if (m_uadpNetworkMessageType == UADPNetworkMessageType.DataSetMessage)
                 {
```

Comment:
Just to make sure this is by intention: the default value for leave open (which is/was) false is now replaced with true.

Reply:
yes, the default passes in a stream  but then it is closed, imho its a implementation error. Caller should be responsible to close the stream.